cst438_register/src/main/java/com/cst438/controller/StudentController.java

```diff
@@ -0,0 +1,92 @@
+package com.cst438.controller;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+import com.cst438.domain.Student;
+import com.cst438.domain.StudentDTO;
+import com.cst438.domain.StudentRepository;
+
+
+@RestController
+public class StudentController {
+	
+	@Autowired
+	StudentRepository studentRepository;
+	
+	@PostMapping("/student")
+	@Transactional
+	public StudentDTO CreateNewStudent(@RequestBody StudentDTO Newstudent) {
```

Comment:
For the REST API, CreateNewStudent, add a couple of comments describing what is happening in your code. For example, what is happening on line 42, why are you calling `createStudentDTO`?